### PR TITLE
XW-144 | Improve critical CSS

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/XW-144

Load critical CSS as a blocking resource and then load the rest as a non blocking one.

### CODE REVIEW

File: front/styles/main/theme/index.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
-@import 'icons';
-@import 'overrides';
+//@import 'icons';
+//@import 'overrides';
+@import 'dark';
```
Comment (rogatty) [inline front/styles/main/theme/index.scss]: Nope, this is loaded lazily in mixins/theme.js

File: front/styles/main/app.scss [hunk 2]
```diff
@@ -1,12 +1,7 @@
-// DO NOT CHANGE:
-// load order of files below is intentional
-@import '../../vendor/wikia-style-guide/src/scss/main';
-@import 'settings';
-@import 'foundation-imports';
-@import 'wikia-variables';
-@import 'mixin/index';
+@import 'shared_imports';
```
Comment (rogatty) [inline front/styles/main/app.scss]: `shared-imports`

File: front/styles/main/component/critical.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@import 'site-head';
```
Comment (rogatty) [inline front/styles/main/component/critical.scss]: This is here and in index.scss, should stay here only.

File: front/styles/main/module/index.scss [hunk 4]
```diff
@@ -22,4 +22,5 @@
 @import 'site-footer';
 @import 'wikia-map';
 @import 'wikia-widget';
-@import 'wiki/index';
+//@import 'wiki/index';
+@import 'wiki/login';
```
Comment (rogatty) [inline front/styles/main/module/index.scss]: What do we do with wiki/index.scss? Right now it's not used but still exists. Maybe it should consist of this single line?

Comment (vforge) [inline front/styles/main/module/index.scss]: `wiki/index.scss` has been removed

File: front/styles/main/component/critical.scss (hunk 3 above)
Comment (vforge) [inline front/styles/main/component/critical.scss]: Imports has been fixed (share-header extends site-head)

**Pull request closed without merging**